### DISCUSSION TITLE
SFTP: Unable to store files

### DISCUSSION THREAD

**Discussion by e-lorden** [GetPublii/Publii#1848] [Help] [answered]
When I try to publish my website in Hostinger via SFTP I get this error.

![image](https://github.com/user-attachments/assets/9d63f7b0-53a1-4f7e-83c2-a0b0927afabe)

And I see that this question has already been raised by others. And without solution.

Why does this happen and how can it be solved?

P.S: All parameters are correct, including the route. In fact, I can connect via Filezilla without any problems. And I have also verified that the path has write permissions.

**Comment by michaelfluegge** [comment, marked as answer]
Hi,
from my experience: 
- the folder on the ftp server exists? (It has to before uploading files with Publii)
- I am on hosteurope, and there you have to be sure to use the right user type  to create the folder
- And the user you use to upload in Publii has to be the user who creates the folder
- the Remote path has a / at the beginning and the end? 
- double check folder/file permissions

**Reply by e-lorden** [reply]
Solved. Thank you very much for your guidance.

**Comment by rickymccallum87** [comment]
The Remote Path field in Publii must be formed like this:

`/home/u[digits]/domains/[unique-website-name].hostingersite.com/public_html`
